orbisdata_plot_with_geopandas.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+
+# Data location
+dirfile = "/home/erenault/geoclimate/osm/output/osm_vannes/"  
+
+# Example of plot for a geojson variable.
+plt.rcParams['figure.figsize'] = (20, 10)
+data = gpd.read_file(dirfile+"block_indicators.geojson")
```

Comment:
Why using a dir ? what about a file path

Reply:
Yes, one can use the **os** module with **import os**.
Given the generic path, for example 'generic_path = /tmp/data/', line 12 becomes:
data = gpd.read_file(os.path.join(generic_path, "block_indicators.geojson")